testing/integration/upgrade_fleet_test.go

```diff
@@ -215,14 +215,19 @@ func testUpgradeFleetManagedElasticAgent(
 	require.NoError(t, err)
 
 	if unprivileged {
-		if startParsedVersion.Less(*upgradetest.Version_8_13_0) {
-			t.Skipf("Starting version %s is less than 8.13 and doesn't support --unprivileged", startParsedVersion.String())
+		if startParsedVersion.Less(*upgradetest.Version_8_13_0_SNAPSHOT) {
+			t.Skipf("Starting version %s is less than 8.13-SNAPSHOT and doesn't support --unprivileged", startParsedVersion.String())
 		}
-		if endParsedVersion.Less(*upgradetest.Version_8_13_0) {
-			t.Skipf("Ending version %s is less than 8.13 and doesn't support --unprivileged", endParsedVersion.String())
+		if endParsedVersion.Less(*upgradetest.Version_8_13_0_SNAPSHOT) {
+			t.Skipf("Ending version %s is less than 8.13-SNAPSHOT and doesn't support --unprivileged", endParsedVersion.String())
 		}
 		if runtime.GOOS != define.Linux {
-			t.Skip("Unprivileged mode is currently only supported on Linux")
+			if startParsedVersion.Less(*upgradetest.Version_8_14_0_SNAPSHOT) {
```

Comment:
```go
if runtime.GOOS != define.Linux {
```

Why not `if runtime.GOOS == define.Windows`? This block could have some confusing error messages if we start testing Macs. At the same time that's likely to be far enough in the future it won't hit this version constraint.